userguide/tasks.rst

```diff
@@ -1137,7 +1137,9 @@ method is selected.
    +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
    | Stream Compression        | SSH       | drop-down menu | Select a compression algorithm to reduce the size of the data being replicated.                                 |
    +---------------------------+-----------+----------------+-----------------------------------------------------------------------------------------------------------------+
-   | Limit (KiB/s)             | SSH       | integer        | Limit replication speed to the specified value in KiB per second. Zero means no limit.                          |
+   | Limit (Ex. 500 KiB/s,     | SSH       | integer        | Limit replication speed to the specified value. Zero means no limit. Differnt units can be specified.           |
```

Comment:
Typo. Also, the examples should use ` :samp: ` markup:
`` :samp:`500 KiB`, :samp:`1M` ``
Also, the entries do not include the "/s", the description should mention that, like "Limit replication speed to this amount per second."
Also, "different" doesn't mean anything with the units.  Maybe
```
Units like :literal:`k`, :literal:`KiB`, and :literal:`M` can be used.
Numbers without unit letters are interpreted as bytes.
```